v0/pipeline/steps/common/dbops-rollback-schema-step-info.yaml

```diff
@@ -20,6 +20,7 @@ allOf:
           type: string
       tag:
         type: string
+        minLength: 1
```

Comment:
shouldnt we remove tag from required fields

Reply:
yes, removed